esbuild.js

```diff
@@ -41,12 +41,12 @@ const buildConfig = {
       out: "./assets/css/qld.bootstrap",
     },
     {
-      in: "./src/js/handlebars.init.js",
+      in: "./src/js/handlebars.helpers.js",
```

Comment:
@stvp-qld

would you not do the IIFE call in the init wrapper?
https://github.com/qld-gov-au/qgds-qol-mvp/blob/develop/src/js/handlebars.init.js

if you are moving init from the path, also delete said file.